### PR TITLE
added Earth Elemental

### PR DESCRIPTION
**Pull request by rahulpandey21**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1444,6 +1444,19 @@ <h1></h1>
 			$(".navbar-collapse").collapse("hide");
 		});
 	</script>
+	
+	 <div class="col-lg-4 mb-4">
```
Comment (vikhyatsingh123) [inline index.html:1448]: Add your code at correct position

**Review by vikhyatsingh123 — CHANGES_REQUESTED** [review]
Insert your code at correct place.

**Comment by rahulpandey21** [comment]
done

File: index.html [hunk 2]
```diff
@@ -1445,16 +1455,7 @@ <h1></h1>
...
-                <div class="card">
-                    <img class="card-img-top" src="https://static.wikia.nocookie.net/slugterra/images/a/af/Earth_Elemental_Slug_Protoform.png/revision/latest/scale-to-width-down/310?cb=20160119164449" alt="Eart Elemental">
-                    <div class="card-body">
-                        <h5 class="card-title">Earth Elemental</h5>
-                        <p class="card-text">The Earth Elemental is one of the Elemental Slugs featured in the Slugterra: Return of the Elementals movie. He has been the first elemental slug that the Shane Gang finds if not including Doc even though he was Ghouled by The Goon. He is one of the few slugs that can still be in velocity without moving .The Earth Elemental use to live in Quartermain Cavern, before being hunted by the Dr. Blakk and The Goon.</p>
-                        <a href="https://github.com/rahulpandey21" class="btn btn-outline-danger btn-sm">Contributed by - Rahul Pandey</a>
-                    </div>
-                </div>
-            </div>
+	 <
```
Comment (vikhyatsingh123) [inline index.html:1458]: Delete opening tag from 1458 line

**Review by vikhyatsingh123 — CHANGES_REQUESTED** [review]

**Review by rahulpandey21 — COMMENTED** [review]
done check it know

**Review by vikhyatsingh123 — COMMENTED** [review]
ok

**Comment by vikhyatsingh123** [comment]
Upload image with smaller size

**Pull request merged**